manifests/implementation/capactio/capact/hello.yaml

```diff
@@ -0,0 +1,53 @@
+ocfVersion: 0.0.1
+revision: 0.1.0
+kind: Implementation
+metadata:
+  prefix: cap.implementation.capactio.capact
+  name: hello
+  displayName: "Hello Capact"
+  description: "Capact Hello Action"
+  documentationURL: https://capact.dev
+  supportURL: https://capact.dev
+  license:
+    name: "Apache 2.0"
+  maintainers:
+    - email: team-dev@capact.dev
+      name: Capact Dev Team
+      url: https://capact.dev
+
+spec:
+  appVersion: "0.3.x"
+  outputTypeInstanceRelations: {}
+
+  implements:
+    - path: cap.interface.capactio.capact.hello
+      revision: 0.1.0
+
+  requires:
+    cap.core.type.platform:
+      oneOf:
+        - name: kubernetes
+          revision: 0.1.0
+
+  imports:
+    - interfaceGroupPath: cap.interface.runner.argo
+      alias: argo
+      methods:
+        - name: run
+          revision: 0.1.0
+    - interfaceGroupPath: cap.interface.capactio.capact
+      alias: capact
+      methods:
+        - name: say
+          revision: 0.1.0
+
+  action:
+    runnerInterface: argo.run
+    args:
+      workflow:
+        entrypoint: hello
+        templates:
+        - name: hello
```

Comment:
Maybe we could include another little step, this time static one, which prints to stdout? In that way we could showcase two things: Interfaces/Implementations and reusability of the smaller "building blocks".